engine/src/test/java/com/redhat/cloud/notifications/EmailTemplatesInDbHelper.java

```diff
@@ -138,25 +135,20 @@ protected String generateAggregatedEmailBody(Action action) {
         return generateEmail(bodyTemplate, action);
     }
 
-    protected String generateEmail(TemplateInstance template, Action action) {
-        String result = template
-            .data("action", action)
-            .data("environment", environment)
-            .data("user", Map.of("firstName", "John", "lastName", "Doe"))
-            .render();
+    @Deprecated
+    protected String generateEmail(TemplateInstance template, Object actionOrEvent) {
 
+        String result = templateService.renderTemplate(createUser(), actionOrEvent, template);
         writeOrSendEmailTemplate(result, template.getTemplate().getId() + ".html");
 
         return result;
     }
 
+    @Deprecated
```

Comment:
Both `generateEmail` methods are now tagged as `Deprecated`.
The new equivalent method isn't it missing ?

Reply:
I guess that's wrong. These shouldn't be deprecated, thanks!